dotcom-rendering/src/web/components/Doughnut.tsx

```diff
@@ -69,6 +79,12 @@ export const Doughnut = ({
 		value: number;
 	}[] = [];
 
+	/**
+	 * These lines help distinguish distinguish segments.
```

Comment:
There is an extra `distinguish` in this comment

Reply:
To make it distinguishable!